src/test/utils/BaseTest.sol

```diff
@@ -202,4 +202,63 @@ abstract contract BaseTest is DSTest, stdCheats {
     function getWallet() public returns (Wallet wallet) {
         wallet = new Wallet();
     }
+
+    function assertBal721(
+        address _token,
+        address _owner,
+        uint256[] memory _tokenIds,
+        bool _assertIsOwner
+    ) internal {
+        for (uint256 i = 0; i < _tokenIds.length; i += 1) {
+            bool isOwnerOfToken = MockERC721(_token).ownerOf(_tokenIds[i]) == _owner;
+
+            if (_assertIsOwner) {
+                assertTrue(isOwnerOfToken);
+            } else {
+                assertTrue(!isOwnerOfToken);
+            }
+        }
+    }
+
+    function assertBal1155Strict(
+        address _token,
+        address _owner,
+        uint256[] memory _tokenIds,
+        uint256[] memory _amounts
+    ) internal {
+        require(_tokenIds.length == _amounts.length, "unequal lengths");
+
+        for (uint256 i = 0; i < _tokenIds.length; i += 1) {
+            assertEq(MockERC1155(_token).balanceOf(_owner, _tokenIds[i]), _amounts[i]);
+        }
+    }
+
+    function assertBal1155(
+        address _token,
+        address _owner,
+        uint256[] memory _tokenIds,
+        uint256[] memory _amounts
+    ) internal {
+        require(_tokenIds.length == _amounts.length, "unequal lengths");
+
+        for (uint256 i = 0; i < _tokenIds.length; i += 1) {
+            assertTrue(MockERC1155(_token).balanceOf(_owner, _tokenIds[i]) >= _amounts[i]);
+        }
+    }
+
+    function assertBal20Strict(
+        address _token,
+        address _owner,
+        uint256 _amount
+    ) internal {
+        assertEq(MockERC20(_token).balanceOf(_owner), _amount);
+    }
+
+    function assertBal20(
```

Comment:
rather than strict or not, we should opt for
`Eq` `Gte` so it'd be more inline with ds-test

Reply:
`assertERC20BalanceEq`
`assertERC20BalanceGte`